src/System.Private.Reflection.Core/src/System/Reflection/Runtime/ParameterInfos/RuntimeParameterInfo.cs

```diff
@@ -43,7 +43,7 @@ public sealed override int GetHashCode()
 
         public abstract override bool HasDefaultValue { get; }
 
-        public sealed override int MetadataToken
+        public override int MetadataToken
```

Comment:
and similar for the other "override int MetadataToken's" - add a comment, or turn into "abstract sealed" - depending how many subclasses would have to be touched as a result...